src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import json
+import random
+import subprocess
+from pathlib import Path
+
+RANDOM_WORDS = [
+    "system",
+    "plug",
+    "gentle",
+    "efficient",
+    "library",
+    "tested",
+    "careful",
+    "sneeze",
+    "excuse",
+    "zoo",
+    "rock",
+    "delight",
+    "hammer",
+    "unit",
+    "happen",
+    "multiply",
+    "texture",
+    "tired",
+    "knot",
+    "yawn",
+]
+
+
+def profile(args):
+    command = (
+        f"perf_analyzer -m {args.model} -i grpc --async --streaming "
+        "--input-data=prompts.json "
+        "--profile-export-file=profile_export.json "
+        "--measurement-mode=count_windows "
+        "--measurement-request-count=10 "
+        "--stability-percentage=999"
```

Comment:
Why do you need a non-default stability %? And why is this value larger than 100?

Reply:
cc @matthewkotila

Reply:
Because of artifacts in how PA was designed, the outcome needs to be considered "stable" in order to dump the timestamps json.

The reason the % is so high rather than default is to likely-guarantee that the outcome is considered "stable".

The reason it's higher than 100% is because of the recent re-definition of stability that you (@nv-braf) came up with like a year and a half or so ago. Basically this is a percent difference between the last 3 passes/windows, which could be higher than 100. I just put 999 as a relatively safe optimization between high-enough-to-not-accidentally-be-unstable and few-enough-characters-to-type-as-to-not-be-annoying-to-manually-type-out.

Reply:
None of this is ideal or ergonomic for how PA was designed, but this uses all of the load manager logic already in PA and is probably easier than trying to rewrite that and come up with some new product and a whole new release process for that.

Reply:
So, this is saying we can deviate by ~10x and still be stable?

Reply:
Something like that, yeah.